linuxkit.yml

```diff
@@ -6,6 +6,7 @@ init:
   - linuxkit/runc:d5cbeb95bdafedb82ad2cf11cff1a5da7fcae630
   - linuxkit/containerd:b6ffbb669248e3369081a6c4427026aa968a2385
   - linuxkit/ca-certificates:67acf038c44bb191ebb704ec7bb39a1524052cdf
+  - linuxkit/wireguard-tools:26fe3d38455f2d441549e3c54bdec1b26ac819b8
```

Comment:
Okay, so it seems then, what you'd like, is for this not to be included in any real `.yml` file, but perhaps just an examples/ file, and then when people actually want to use it, they'll include wireguard-tools themselves?

Or do you not even want this to live in pkgs/ at all, and instead just have people install this from apk from within? This seems like it might be _slightly_ more reasonable, with one small problem being that wireguad-tools is only on Alpine testing.